cmd/pinniped/cmd/kubeconfig.go

```diff
@@ -74,9 +75,13 @@ type getKubeconfigOIDCParams struct {
 
 type getKubeconfigConciergeParams struct {
 	disabled          bool
+	credentialIssuer  string
 	authenticatorName string
 	authenticatorType string
 	apiGroupSuffix    string
+	caBundlePath      string
+	endpoint          string
```

Comment:
We talked about this on Zoom and I think resolved the confusion. The reason it's in the login commands even in the impersonation proxy case is because we want to be able to correctly cache credentials even on older versions of kubectl where we don't get the cluster info passed in.